app/views/foreman_ansible/job_templates/ansible_collections_-_install_from_galaxy.erb

```diff
@@ -0,0 +1,32 @@
+<%#
+name: Ansible Collection - Install from Galaxy
+job_category: Ansible Galaxy
+description_format: Install collections '%{ansible_collections_list}' from Galaxy
+snippet: false
+template_inputs:
+- name: ansible_collections_list
+  required: true
+  input_type: user
+  description: "List of collections in Ansible Galaxy to install, separated by commas, e.g:
+    \"mysql, nginx\"\r\n\r\nThe default location is the 'collections_paths' configured on
+    /etc/ansible/ansible.cfg, you may override it by filling the 'location' input.
+    Click on \"Advanced\" to see it."
+  advanced: false
+- name: location
```

Comment:
Since we've got loc/org options/things mostly in context of taxonomies, I'd suggest to rename the input to `target` or `dir`, or something similar.

Reply:
done